packages/aws-cdk/lib/api/hotswap/ecs-services.ts

```diff
@@ -45,14 +46,18 @@ export async function isHotswappableEcsServiceChange(
   const namesOfHotswappableChanges = Object.keys(classifiedChanges.hotswappableProps);
   if (namesOfHotswappableChanges.length > 0) {
     const taskDefinitionResource = await prepareTaskDefinitionChange(evaluateCfnTemplate, logicalId, change);
+    if (taskDefinitionResource === undefined) {
+      reportNonHotswappableChange(ret, change, undefined, 'Found unsupported changes to the task definition', false);
```

Comment:
Would be nice if we can report the property path that we found unsupported changes in.